### DISCUSSION TITLE
Pin a specific minor version

### DISCUSSION THREAD

**Discussion by gbrindisi** [github-community-projects/evergreen#213] [Q&A]
Hi,
I am a bit confused and looking for some clarification.

I target specifically a minor version in my setup, like this:
```
- id: run-evergreen
        name: Run evergreen action
        uses: github/evergreen@v1.12.0
```

and I noticed that it still always end up puling the latest major version (from a run log):
```
Download action repository 'github/evergreen@v1.12.0' (SHA:8d2d4117eadbbfa4c33828364cc4c88fea908192)
...
Pull down action image 'ghcr.io/github/evergreen:v1'
```

But in the registry I do see that there are tags with the minor versions, for example https://github.com/github/evergreen/pkgs/container/evergreen/260302251?tag=v1.12.0 

So what should I do to pin evergreen to a specific minor v?

cc @zkoppert

**Comment by zkoppert** [comment]
oh darn. This might be because when you specify the action at v1.12.0 it goes to the code at that point and determines which package to pull. See here the v1.12.0 of the code that specifies the package: https://github.com/github/evergreen/blob/v1.12.0/action.yml#L7

You can see it just goes to the registry and pulls the v1 tag of the container. We will need some automation to get that updated to the specific container version every time a release occurs.

**Reply by gbrindisi** [reply]
ah good to know! I will create an issue :+1:

**Reply by gbrindisi** [reply]
Here you go: https://github.com/github/evergreen/issues/221

**Comment by hkadakia** [comment]
Here's a workaround that worked for me:
```
- id: run-evergreen
   name: Run evergreen action
   uses: docker://ghcr.io/github/evergreen:v1.14.0
```

**Reply by gbrindisi** [reply]
thanks!
